_data/navigation.js

```diff
@@ -107,6 +107,12 @@ module.exports = {
           {
             title: 'Feed Registry',
             url: '/docs/feed-registry/',
+            children: [
+              {
+                title: 'Feed Registry Functions',
+                url: '/docs/feed-registry-functions/'
+              }
+            ]
           },
           {
             title: 'API Reference',
```

Comment:
For consistency , I would put this "API Reference" as child of "Using Data Feeds". Hence it will be clear in our docs that : 

- Data Feeds through proxy or aggregators have a certain API Reference
- Data FEeds through feed registry have a certain API Reference